### PR TITLE
Maven plugin: exposed the settings 'node.version' and 'node.download.…

### PR DESCRIPTION
**Pull request by mvysny**
…root'

Related issue: #8603

### CODE REVIEW

**Comment by mvysny** [comment]
@pleku thank you, good catch! I've fixed the PR accordingly.

**Comment by pleku** [comment]
Tests fail due to `null` string for the URL https://teamcity.vaadin.com/viewLog.html?buildId=64657&buildTypeId=Flow_FlowTests&tab=buildResultsDiv

**Comment by mvysny** [comment]
Thank you, fixed :+1:

File: flow-maven-plugin/src/main/java/com/vaadin/flow/plugin/maven/FlowModeAbstractMojo.java [hunk 1]
```diff
@@ -143,7 +143,7 @@ public boolean useDeprecatedV14Bootstrapping() {
      * Vaadin-default node version - see {@link FrontendTools} for details.
      */
     @Parameter(property = "node.version", defaultValue = FrontendTools.DEFAULT_NODE_VERSION)
-    protected String nodeVersion;
+    protected String nodeVersion = FrontendTools.DEFAULT_NODE_VERSION;
```
Comment (pleku) [inline flow-maven-plugin/src/main/java/com/vaadin/flow/plugin/maven/FlowModeAbstractMojo.java:146]: I think the correct fix for the unit tests should have been to use e.g.

```java
ReflectionUtils.setVariableValueInObject(mojo, "node.version", FrontendTools.DEFAULT_NODE_VERSION;
```
and the same for the "nodeDownloadRoot" for `BuildFrontendMojoTest` and `PrepareFrontendMojoTest`

Comment (mvysny) [reply flow-maven-plugin/src/main/java/com/vaadin/flow/plugin/maven/FlowModeAbstractMojo.java]: Sure thing, thanks!

**Pull request merged**